zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/cache/ProcessCacheMetrics.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.cache;
+
+import com.github.benmanes.caffeine.cache.RemovalCause;
+import com.github.benmanes.caffeine.cache.stats.CacheStats;
+import com.github.benmanes.caffeine.cache.stats.StatsCounter;
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Timer;
+import java.util.concurrent.TimeUnit;
+
+public class ProcessCacheMetrics implements StatsCounter {
+
+  private static final String NAMESPACE = "zeebe.camunda.exporter.processcache";
+
+  private final Counter hitCount;
+  private final Counter missCount;
+  private final Timer loadSuccessDuration;
+  private final Timer loadFailureDuration;
+  private final Counter evictionCount;
+
+  public ProcessCacheMetrics(final MeterRegistry meterRegistry) {
```

Comment:
:x: We should add labels for the partition

Reply:
This is actually done already in the context https://github.com/camunda/camunda/blob/main/zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/context/ExporterContext.java#L49-L54